src/tls13.c

```diff
@@ -4075,6 +4266,135 @@ static int Dtls13DoDowngrade(WOLFSSL* ssl)
 }
 #endif /* WOLFSSL_DTLS13 && !WOLFSSL_NO_CLIENT*/
 
+/* check if the server accepted ech or not */
+#if defined(HAVE_HPKE) && defined(HAVE_ECC)
+static int EchCheckAcceptance(WOLFSSL* ssl, const byte* input,
+  int serverRandomOffset, int helloSz)
+{
+    int ret = 0;
+    int digestType;
+    int digestSize;
+    HS_Hashes* tmpHashes;
+    HS_Hashes* acceptHashes;
+    byte zeros[WC_SHA512_DIGEST_SIZE] = {0};
```

Comment:
WC_MAX_H=DIGEST_SIZE in hash.h is the maximum supported by configuration. Use that instead.

Reply:
did you mean `WC_MAX_DIGEST_SIZE`? I will use that

Reply:
Yes. Delete is next to equal apparently!